code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -9,6 +9,11 @@
 	large = 1
 	ventcrawler = 0
 	pressure_resistance = 200 //Because big, stompy xenos should not be blown around like paper.
+	move_resist = MOVE_FORCE_STRONG //Yes, queenos is huge and heavy
+	alien_disarm_damage = 60 //Queens do higher disarm stamina damage than normal aliens
+	alien_slash_damage = 30 //Queens do higher slashing damage to people
+	var/alien_queen_movement_delay = 1 //This represents a movement delay of 1, or roughly 80% the movement speed of a normal carbon mob
```

Comment:
this should probably be another `/mob/living/carbon/alien/humanoid` level var

Reply:
Addressed